src/logic/operate.js

```diff
@@ -0,0 +1,40 @@
+const Big = require('big.js');
+
+const operate = (ValueOne, ValueTwo, opt) => {
+  let result;
+  if (opt === '+') {
+    result = Big(ValueOne).plus(ValueTwo);
+  }
+
+  if (opt === '+/-') {
+    result = Big(ValueOne).plus(-1);
+  }
```

Comment:
This operator should be changing a positive number to a negative and vice versa, you have done this in the `calculate.js` file, either remove it here or change the logic.

Reply:
I fixed this issue moving to operate the logic